work_queue/src/bindings/python3/PyTask/PyTask.py

```diff
@@ -0,0 +1,107 @@
+import work_queue as wq
+import dill
+import os
+import sys
+import tempfile
+import uuid
+import traceback
+
+
+class PyTask(object):
```

Comment:
Also, I think you want the PyTask object to extend the Task object, so that it supports all the usual functions to set resources, add additional files, etc...